spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/security/saml2/Saml2RelyingPartyRegistrationConfiguration.java

```diff
@@ -78,6 +79,8 @@ private RelyingPartyRegistration asRegistration(String id, Registration properti
 				(details) -> details.binding(properties.getIdentityprovider().getSinglesignon().getBinding()));
 		builder.providerDetails((details) -> details.signAuthNRequest(signRequest));
 		builder.credentials((credentials) -> credentials.addAll(asCredentials(properties)));
+		Optional.ofNullable(properties.getLocalEntityIdTemplate())
```

Comment:
If we give the property a default value as proposed above, this can always be applied without checking if it's `null`.